style.css

```diff
@@ -0,0 +1,113 @@
+body {
+  background-color: #f6eada;
+  color:#128057
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Jost", sans-serif;
+}
+
+.filter-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 50px;
+}
+
+header {
+  text-align: center;
+  font-size: 2rem;
+  font-weight: 500;
+  background-color: #ee4574;
+}
+
+#title {
+  margin-top: 2rem;
+}
+
+.gallery-wrapper {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  column-gap: 35px;
+  row-gap: 20px;
+  margin: 40px 40px 40px 40px;
+} 
+
+.card-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  max-width: 100%;
+  padding: 16px, 16px, 24px, 16px;
+  box-sizing: border-box;
+  border-radius: 20px;
+  border: dotted #ee4574;
+  cursor: pointer; 
+}
+
+.card-wrapper:hover {
+  border: solid;
+  border-radius: 20px;
+  border-color: #128057;
+  font-size: 1.5rem;
+}
+
+img {
+  max-width: 100%;
+  aspect-ratio: 5 / 4;
+  border-radius: 10px;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.detail-content {
+  margin-bottom: 1rem;
+  margin-left: 0.5rem;
+}
+
+span {
+  font-weight: 600;
+}
+
+button {
+  border-radius: 20px;
+  padding: 0 10px;
+  margin: 0 5px;
+  cursor: pointer;
+  background-color: #128057;
+  color: #ee4574;
+}
+
+button:hover {
+  background-color: #ee4574;
+  color: #128057 ;
+}
+
+header:hover{
+  cursor: pointer;
+}
+
+@media only screen and (max-width:500px) {
+  .gallery-wrapper {
+    display: flex;
+    flex-direction: column;  
+  }
+  .card-wrapper {
+    .card-wrapper:hover{
+      display: none;
```

Comment:
maybe put the .card-wrapper:hover{} outside of the card-wrapper? 

but also a question: why do you want the card to disappear on hover for smaller screens?